backend/app/services/session_turn_service.py

```diff
@@ -37,6 +41,33 @@ def get_audio_content_type(upload_file: UploadFile) -> str:
     return mime
 
 
+def store_audio_file(session_id: UUID, audio_file: UploadFile) -> str:
+    audio_name = f'{session_id}_{uuid4().hex}'
+    gcs = GCSManager('audio')
```

Comment:
Please initialize the `GCSManager` only once, so it can be re-used between multiple API calls from multiple endpoints. It improves performance and should be fine as all API endpoints we have are stateless (AFAIK at least).